frontend/public/components/storage-class.tsx

```diff
@@ -12,7 +12,7 @@ export const StorageClassReference: K8sResourceKindReference = 'StorageClass';
 const { common } = Kebab.factory;
 const menuActions = [...Kebab.getExtensionsActionsForKind(StorageClassModel), ...common];
 
-const defaultClassAnnotation = 'storageclass.kubernetes.io/is-default-class';
+export const defaultClassAnnotation = 'storageclass.kubernetes.io/is-default-class';
```

Comment:
Don't export this, use `isDefaultClass` below

Reply:
ack